sunpy/database/tests/test_attrs.py

```diff
@@ -422,7 +422,7 @@ def test_walker_create_vso_instrument(vso_session):
                              wavemax=7.293188082141598e-05)]
     assert len(entries) == len(expected)
     for e in entries:
-        assert e in entries
```

Comment:
No wonder this test never failed.

Reply:
Is this on master too?

Reply:
yes https://github.com/sunpy/sunpy/blob/master/sunpy/database/tests/test_attrs.py#L425

Reply:
Both of them are. No wonder they didn't fail.